akka-stream-tests/src/test/scala/akka/stream/io/FileSinkSpec.scala

```diff
@@ -129,8 +141,9 @@ class FileSinkSpec extends AkkaSpec(UnboundedMailboxConfig) {
 
   }
 
-  private def targetFile(block: File ⇒ Unit) {
+  private def targetFile(block: File ⇒ Unit, create: Boolean = true) {
     val targetFile = File.createTempFile("synchronous-file-sink", ".tmp")
+    if (!create) targetFile.delete()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/vasseurjb/akka/file/2046285078/issues/source?bid=3132306&fileBranchId=3132378#l147)